app/angular/package.json

```diff
@@ -51,7 +51,8 @@
     "@angular/platform-browser": ">=6.0.0",
     "@angular/platform-browser-dynamic": ">=6.0.0",
     "babel-loader": "^7.0.0 || ^8.0.0",
-    "zone.js": "^0.8.27"
+    "zone.js": "^0.8.27",
```

Comment:
Sorry for the late response - due to a version update your zone.js change now produces a conflict. Can you fix this please? I think you just need to set `^0.8.29`